Docs/user_guide/modules/surfacetoolbox.md

```diff
@@ -31,6 +32,19 @@ These tools can be combined for multiple surface model processing effects.
 This tool can merge coincident points, remove unused points (i.e. not used by
 any cell) and treat degenerate cells.
 
+### Uniform remeshing
+
+Uniformly remesh the surface using [ACVD algorithm](https://github.com/pyvista/pyacvd).
+
+This resampling typically provides higher quality meshes than decimation, with similar computation time.
+
+- **Subdivide:** Number of subdivision to perform before remeshing. Each subdivision creates 4 triangles for each input triangle. This is needed if the required number of clusters are higher than the number of cells in the input mesh.
+- **Clusters:** Number of desired cells in the output mesh. Use higher number to preserve more details.
```

Comment:
I've made a mistake when documenting this parameter (I'll fix that in a couple of minutes) in Surface toolbox. Clusters actually means the number of desired points (not cells).

```suggestion
- **Number of points:** Number of desired points in the output mesh. Use higher number to preserve more details.
```